src/mobile/apps/feature/index.coffee

```diff
@@ -15,14 +15,10 @@ app.get '/feature/:id', routes.index
 app.get '/feature/:id/:queryParams', routes.index
 
 app.get '/auction/:id/bid/:artworkId', routes.bid
+
 # TODO: as of 7/15/19 none of these confirm-registration routes
 # have been called for > 30 days. Investigate removal.
 app.get '/artwork/:id/confirm-registration', routes.confirmRegistration('artwork')
-app.get '/auction/:id/bid/:artworkId/confirm-registration', routes.confirmRegistration('bid')
 
-# Legacy route support for Eigen
-app.get '/feature/:id/bid/:artworkId', routes.bid
-app.get '/feature/:id/bid/:artworkId/confirm-registration', routes.confirmRegistration('bid')
 # Also:
 app.get '/auctions/:id/bid/:artworkId', routes.bid
-app.get '/auctions/:id/bid/:artworkId/confirm-registration', routes.confirmRegistration('bid')
```

Comment:
Are any of these URLs used as a direct link anywhere else (e.g. marketing emails)? I wonder if we should leave them for a while and perform redirections.

Reply:
I checked in datadog, but this route is specifically the bid+register confirmation modal so can't be triggered from anywhere else.